lib/internal.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+const { readFile } = require('fs/promises');
+const { getMHTDocument } = require('./utils');
+const getDocumentTemplate = require('./templates/document');
+
+module.exports = {
+    generateDocument,
+    addFiles,
+};
+
+async function generateDocument(zip) {
```

Comment:
Let's add docblocks with param types and descriptions to all functions (the same for utils).

Reply:
Good idea

Reply:
done